applications/sasquatch/values.yaml

```diff
@@ -283,3 +283,10 @@ global:
   # -- Base path for Vault secrets
   # @default -- Set by Argo CD
   vaultSecretsPath: ""
+
+app-metrics:
```

Comment:
I think the Helm style uses camelCase for variables, so this would be `appMetrics`?

Reply:
Oh right. Good call. I guess using the sub chart name is necessary to map to its own values.